website/content/en/docs/deployment/cognito-rds-s3/guide.md

```diff
@@ -24,18 +24,8 @@ Refer to the [general prerequisites guide]({{< ref "/docs/deployment/prerequisit
     1. Create TLS certificates for the domain
     1. Create a Cognito Userpool
     1. Configure Ingress
-2. Deploy Kubeflow. Choose one of the two options to deploy kubeflow:
-    1. **[Option 1]** Install with a single command:
-        {{< tabpane persistLang=false >}}
-        {{< tab header="Kustomize" lang="toml" >}}
-        make
-        {{< /tab >}}
-        {{< tab header="Helm" lang="yaml" >}}
-        make
-        {{< /tab >}}
-        {{< /tabpane >}}
-
-    1. **[Option 2]** Install individual components:
+2. Deploy Kubeflow.
```

Comment:
Looks like my comment was not clear. Apologies for miscommunication.

We shall remove the individual command installation and have only install with single command like other options

Reply:
There is no single command installation option for cognito-rds-s3 afaik?